plugin/src/withSentry.ts

```diff
@@ -0,0 +1,70 @@
+import type { ConfigPlugin } from 'expo/config-plugins';
+import { createRunOncePlugin, WarningAggregator } from 'expo/config-plugins';
+
+import { withSentryAndroid } from './withSentryAndroid';
+import { withSentryIOS } from './withSentryIOS';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const pkg = require('../../package.json');
+
+interface PluginProps {
+  organization?: string;
+  project?: string;
+  authToken?: string;
+  url?: string;
+}
+
+const withSentry: ConfigPlugin<PluginProps | void> = (config, props) => {
+  const sentryProperties = getSentryProperties(props);
+  let cfg = config;
+  if (sentryProperties !== null) {
+    try {
+      cfg = withSentryAndroid(cfg, sentryProperties);
+    } catch (e) {
+      WarningAggregator.addWarningAndroid(
+        'sentry-expo',
```

Comment:
it looks like we left some sentry-expo references in here, should update to this lib name